module/FlowManagement/src/FlowManagement/FlowCard.php

```diff
@@ -0,0 +1,70 @@
+<?php
+
+namespace FlowManagement;
+
+use Application\DomainEntity;
+use Rhumsaa\Uuid\Uuid;
+use Application\Entity\BasicUser;
+
+class FlowCard extends DomainEntity implements FlowCardInterface {
+
+	/**
+	 * @var BasicUser
+	 */
+	private $recipient;
+	/**
+	 * @var array
+	 */
+	private $content;
+	/**
+	 * @var \DateTime
+	*/
+	protected $createdAt;
+	/**
+	 * @var BasicUser
+	 */
+	protected $createdBy;
+	/**
+	 * @var \DateTime
+	 */
+	protected $mostRecentEditAt;
+	/**
+	 * @var BasicUser
+	 */
+	protected $mostRecentEditBy;
+
+	public static function create(BasicUser $recipient, $content, BasicUser $by){
+		$rv = new self();
+		$event = FlowCardCreated::occur(Uuid::uuid4()->toString(), [
+				'to' => $recipient->getId(),
+				'content' => $content,
+				'by' => $by->getId()
+		]);
+		$rv->recordThat($event);
+		return $rv;
+	}
+
+	protected function whenFlowCardCreated(FlowCardCreated $event) {
```

Comment:
Sposto nella classe specifica solamente la "specializzazione" della variabile `$this->content`